unitary/alpha/quantum_world.py

```diff
@@ -163,6 +261,21 @@ def _suggest_num_reps(self, sample_size: int) -> int:
             sample_size = 100
         return sample_size
 
+    def _interpret_result(self, result: Union[int, Iterable[int]]) -> int:
+        """Canonicalize an entry from the measurement results array to int."""
```

Comment:
Optional.  Consider explaining what "canonicalizing" is, in this context (ie. that it is really just for qudits)

Reply:
Done